src/lib/pickles/unfinalized.ml

```diff
@@ -71,8 +71,23 @@ module Constant = struct
          Plonk_types.Evals.to_in_circuit Dummy.evals_combined.evals.evals
        in
        let env =
+         let module Env_bool = struct
+           type t = bool
+
+           let true_ = true
+
+           let false_ = false
+         end in
+         let module Env_field = struct
+           include Tock.Field
+
+           type bool = Env_bool.t
+
+           let if_ (b : bool) ~then_ ~else_ = if b then then_ () else else_ ()
```

Comment:
Do we know each other, `Env_field` based on `Tock.field` ? Looks like we already met 😝